### PR TITLE
KLBP-36: Styling confirmation page.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/assets/images/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import logo from './logo.svg'
+import spinner from './spinner.svg'
+import confirmationIcon from './confirmationIcon.svg'
+
+export { logo, spinner, confirmationIcon }
```
Comment (ioan-iacob-netea) [inline src/assets/images/index.ts:5]: Please order alphabetically the import and export elements.

File: src/components/Confirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <Spinner size="w-24 h-24" className="mx-auto" />
+      </div>
+    )
+  }
+
+  return (
+    <div className="font-bold">
+      <img src={confirmationIcon} alt="accept" className="mb-10 inline-block" />
+      <h2 className="text-4.5xl leading-tight">
+        Nous vous remercions
```
Comment (ioan-iacob-netea) [inline src/components/Confirmation.tsx:22]: Please use translations for texts outside tests.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/components/Confirmation.tsx (hunk 2 above)
Comment (iuliudascalu) [reply src/components/Confirmation.tsx]: Done.

File: src/assets/images/index.ts (hunk 1 above)
Comment (iuliudascalu) [reply src/assets/images/index.ts]: Done.

File: src/lang/fr.json [hunk 3]
```diff
@@ -6,5 +6,9 @@
   "homepage.text3": "Soyez rassuré, vos données resteront protégées et confidentielles, conformément à la",
   "homepage.text3.anchor": " Politique de Protection des Données Personnelles de La Banque Postale.",
   "homepage.text4": "(1) art. R561-5 et R561-12 du Code Monétaire et Financier",
-  "homepage.welcome": "Bienvenue!"
+  "homepage.welcome": "Bienvenue!",
+  "confirmation.page.title": "Nous vous remercions",
+  "confirmation.page.subtitle": "pour votre temps !",
+  "confirmation.page.description.1": "Vos informations et documents ont bien été transmis à La Banque Postale et sont en cours d’analyse par nos équipes.",
+  "confirmation.page.description.2": "Vous pouvez désormais quitter cette page."
```
Comment (ioan-iacob-netea) [inline src/lang/fr.json:13]: Please order the translations.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/lang/fr.json (hunk 3 above)
Comment (iuliudascalu) [reply src/lang/fr.json]: Done.

**Review by ioan-iacob-netea — APPROVED** [review]

**Review by emilomirpitechplus — APPROVED** [review]

**Review by andreir-suciu — APPROVED** [review]

**Review by mariuspop86 — APPROVED** [review]

**Pull request merged**